src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: ../../layouts/ContentLayout.astro
+---
+
+# Ce qui caractérise le top 20% des devs
+
+## Introduction
```

Comment:
You can remove the `## Introduction`: it add no value :).

Reply:
Done. 🙂